app/business_model/models.py

```diff
@@ -57,6 +57,7 @@ def default_fate_values(self):
 
 class BMQuestion(models.Model):
     question_for_user = models.TextField(null=False)
+    sub_question_to = models.IntegerField(null=True)
```

Comment:
This feature is not being used yet, right? Or am I missing it?

Reply:
indeed it is not used yet, only for future use